src/utils/attributionHtml.js

```diff
@@ -20,7 +20,7 @@ function attributionHtml(image, ccLicenseURL, fullLicenseName) {
     ).join('');
   }
 
-  const licenseImgLink = `<a href="${ccLicenseURL}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;opacity: .7;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`;
+  const licenseImgLink = `<a href="${ccLicenseURL}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;opacity: 1.0;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`;
```

Comment:
Instead of hard-coding a `opacity: 1.0`, it's better to just remove the `opacity` style. People using this on their websites might have their own opacity value on their CSS, and this will override it